### ISSUE TITLE
Order layout panel should be enabled only when we are focused on that panel

### ISSUE THREAD

**Issue by ichim-david** [plone/volto#6492]
**Describe the bug**
The order side panel is rendering even when it's the order button isn't focused.
We should enable it only when we want to interact with it as it's another set of elements that re-render every time
there is a change in the blocks within the page.
**To Reproduce**
Use devtools to see the order panel logic rendered even when you don't intend to interact with it because your focus
is on the page options or a certain focused block.

**Expected behavior**
Order panel should render only when focused.

**Comment by ichim-david** [comment]
@Deathnote45 there are some expectations that we require for open source contributors to follow before contributing. Have a look at the docs here: https://6.docs.plone.org/contributing/first-time.html

**Comment by stevepiercy** [comment]
@Deathnote45 please read and follow [First-time contributors](https://6.docs.plone.org/contributing/first-time.html), especially [Things not to do](https://6.docs.plone.org/contributing/first-time.html#things-not-to-do), [Contributing to Plone](https://6.docs.plone.org/contributing/index.html), and [Contributing to Volto](https://6.docs.plone.org/contributing/volto.html).

**Comment by saurabhraghuvanshii** [comment]
Hii @ichim-david is anyone working on it. if not I like to work on this.

**Comment by kaushik7kk** [comment]
Hi @ichim-david ! I would like to work on this issue if it's not already being dealt with. Thanks!

**Comment by saurabhraghuvanshii** [comment]
@ichim-david sorry sir for inconvenience, deeply sorry, it's my fault for not reading yours contribution guide properly.

**Comment by stevepiercy** [comment]
@ichim-david we discussed already that we want a clean comment history, and not scroll through noise. I used to leave my own admonitions, but they were ignored repeatedly. Going forward, we can leave the first admonition and delete the rest.

However in all of my experience, those who post a comment such as the one above don't read _anything_ except the title, and maybe the description, of the issue. That's why I have https://docs.github.com/en/get-started/writing-on-github/working-with-saved-replies/creating-a-saved-reply to make it easier to admonish such inconsiderate behavior. I almost never bother to personalize these messages because they don't bother to read a readme or contributing guide. Here's the one I use most often, and I delete the Volto bit for other repos.

```markdown
please read and follow [First-time contributors](https://6.docs.plone.org/contributing/first-time.html), especially [Things not to do](https://6.docs.plone.org/contributing/first-time.html#things-not-to-do), [Contributing to Plone](https://6.docs.plone.org/contributing/index.html), and [Contributing to Volto](https://6.docs.plone.org/contributing/volto.html).
```

**Comment by nitinawari** [comment]
@ichim-david  As I understand it, you're saying that when we open the layout tab, it currently defaults to rendering the order tab inside instead of the block tab. Are you suggesting that we should render the block tab by default instead of the order tab? Please let me know if I misunderstood

**Comment by ichim-david** [comment]
@nitinawari no. What I mean is this, the order tab renders it's content which is a drag and drop list of blocks that mirrors the drag and drop seen on the main edit area.
The order Component should only be rendered when the Order tab is activated, otherwise it should return null or '' so that it doesn't influence the performance of the editing interface since again it mirrors the editing interface blocks.

**Comment by kaushik7kk** [comment]
@nitinawari I think @ichim-david is talking strictly about the markup part which you can see by inspecting the page and it renders even when you have selected the other tabs. So, it should only be rendered when the Orders tab is active and we want to see its contents and not always by default.

**Comment by Abhishek-17h** [comment]
@ichim-david I’ve resolved the issue with the orderTab, but I noticed a new issue after that fix. The elements of the "Order" tab now take a long time to render after clicking on it. This delay seems to be related to the activeTab state and how it’s managed when switching tabs. It appears that the tab content is not updating immediately, causing the delay. Do u have any suggestions for this  ?
[Screencast from 2024-12-25 22-03-41.webm](https://github.com/user-attachments/assets/5df415cf-c5ad-441e-a452-aab58019228a)

**Comment by Abhishek-17h** [comment]
No problem. Thank you for your replay and enjoy 😊.

**Comment by dev-palwar** [comment]
@ichim-david  Instead of repeatedly reminding users to read the guide, a GitHub workflow or bot can be set up to automatically detect such comments and reply with 'Please read the guide!'. Happy to help with this if needed.

**Comment by stevepiercy** [comment]
@dev-palwar GitHub does not have an API event endpoint for comments in an existing issue or pull request from first-timers. It does have an API event endpoint when a first-timer opens an issue or pull request. I have such a bot, [`boring-cyborg`](https://github.com/plone/documentation/blob/6.0/.github/boring-cyborg.yml) enabled in `plone/documentation`.

Other than that, parsing comments for keywords may be possible, but language context is tricky. You're welcome to try, but you should open a new issue to discuss it further.

**Comment by Abhishek-17h** [comment]
@ichim-david can u now have a look on this issue about order tab rendering that I mentioned above?
